wezterm-gui/src/glyphcache.rs

```diff
@@ -3277,117 +3277,117 @@ impl BlockKey {
             ]),
 
             // Upper half block
-            0x2580 => Self::Upper(4),
+            0x2580 => Self::Upper(4), // ▀
             // Lower 1..7 eighths
-            0x2581..=0x2587 => Self::Lower((c - 0x2580) as u8),
-            0x2588 => Self::Full(BlockAlpha::Full),
+            0x2581..=0x2587 => Self::Lower((c - 0x2580) as u8), // ▁ ▂ ▃ ▄ ▅ ▆ ▇
+            0x2588 => Self::Full(BlockAlpha::Full), // █
             // Left 7..1 eighths
-            0x2589..=0x258f => Self::Left((0x2590 - c) as u8),
+            0x2589..=0x258f => Self::Left((0x2590 - c) as u8), // ▉ ▊ ▋ ▌ ▍ ▎ ▏
             // Right half
-            0x2590 => Self::Right(4),
-            0x2591 => Self::Full(BlockAlpha::Light),
-            0x2592 => Self::Full(BlockAlpha::Medium),
-            0x2593 => Self::Full(BlockAlpha::Dark),
-            0x2594 => Self::Upper(1),
-            0x2595 => Self::Right(1),
-            0x2596 => Self::Quadrants(Quadrant::LOWER_LEFT),
-            0x2597 => Self::Quadrants(Quadrant::LOWER_RIGHT),
-            0x2598 => Self::Quadrants(Quadrant::UPPER_LEFT),
-            0x2599 => {
+            0x2590 => Self::Right(4), // ▐
+            0x2591 => Self::Full(BlockAlpha::Light), // ░
```

Comment:
```suggestion
            // ░
            0x2591 => Self::Full(BlockAlpha::Light),
```

Reply:
So I add a comment line above ALL the oneline glyphs overrides?

Reply:
I think the comment works better when it precedes the codepoint value, as it is in the same horizontal position, which makes it easier to visually scan.

In hindsight, I should have put the codepoint description there for all of those glyphs when I started adding them; it didn't feel necessary when I started, but it definitely made sense when I did the box drawing glyphs.

If you're up for automating adding those in, that would be fantastic, but it's not required in this PR!

Reply:
I can automate it quite easily if I have a list of all the codepoint descriptions and their corresponding 0x1234 notation, find me that and I'll do it in this PR 🙃